store/index.ts

```diff
@@ -72,27 +85,32 @@ export const mutations: MutationTree<RootState> = {
   },
   SOCKET_RECONNECT_ERROR(state: RootState) {
     Vue.set(state.socket, 'reconnectError', true);
+  },
+  SET_SYNC_STATUS(state: RootState, value: boolean) {
+    state.syncing = value;
   }
 };
 
 export const actions: ActionTree<RootState, RootState> = {
   async reset({ dispatch }, { clearCritical }: { clearCritical: boolean }) {
     const promises = [];
     promises.push(dispatch('backdrop/clearAllBackdrop', { root: true }));
-    promises.push(dispatch('deviceProfile/clearDeviceProfile', { root: true }));
```

Comment:
Why this one isn't cleared now?

Reply:
deviceID, deviceName, clientVersion and clientName don't change at any point, since they're all linked to either the hardware or the client.

As such, it makes no sense to clean them when logging out. In fact, they should be initialized in Nuxt's `nuxtServerInit` hook anyway, but that can be done later (it works for now)

Reply:
I'd still add them in the clearCritical if, just in case we need it later (the if case isn't used for now, it fills the same use case IMO)